osquery/tables/system/linux/pci_devices.cpp

```diff
@@ -392,10 +393,20 @@ QueryData genPCIDevices(QueryContext& context) {
     return results;
   }
 
-  std::ifstream raw(kPciidsPath);
+  std::ifstream raw;
+  std::string all_path;
+  for (const std::string& kPciidsPath : kPciidsPathList) {
```

Comment:
We use the `kPciidsPathList` naming pattern for constants, you can write this as
```cpp
for (const std::string& pci_ids_path : kPciidsPathList) {
```